### DISCUSSION TITLE
How to specify collation for MongoDB collection?

### DISCUSSION THREAD

**Discussion by chunhualiu** [mikro-orm/mikro-orm#2230] [Q&A]
I want the entity to be created with a specified collation.
I could not find a way to do it.
And I checked the source code for createCollection, seems it did not pass the options.
`
  async createCollection(name: EntityName<AnyEntity>): Promise<Collection> {
    return this.db.createCollection(this.getCollectionName(name));
  }
`
So is there a way to do it?

**Comment by B4nan** [comment]
We can add the options parameter, no problem with that (PR welcome).

**Reply by chunhualiu** [reply]
Thank you :)
